docs/reference/api.md

```
@@ -395,6 +395,172 @@ Removes cached images not referenced by config or any existing shed.
+      "docker_ref": "ghcr.io/example/default:v1",
+      "size": {"logical_bytes": 5368709120, "physical_bytes": 4831838208}
+    },
+    {
+      "name": "_base",
+      "path": "/var/lib/shed/vz/_base-rootfs.ext4",
+      "size": {"logical_bytes": 5368709120, "physical_bytes": 0},
+      "is_base": true
+    }
+  ],
+  "kernel": {
+    "path": "/var/lib/shed/vz/vmlinux",
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)